crates/store/re_types/src/datatypes/pixel_format_ext.rs

```diff
@@ -70,12 +160,56 @@ impl PixelFormat {
         }
     }
 
+    /// Returns true if the format is a YUV format using
+    /// limited range YUV.
+    ///
+    /// I.e. for 8bit data, Y is valid in [16, 235] and U/V [16, 240], rather than 0-255.
+    pub fn is_limited_yuv_range(&self) -> bool {
```

Comment:
I think this would be nicer as `fn yuv_range(&self) -> Option<YuvRange>` so we can return `None` for non-yuv formats in the future. Also, enums is nicer than bools.

Reply:
yeah but I didn't want to introduce this enum a third time (1x each in re_renderer, re_types and re_video -.-)

Reply:
If it was a good idea in the other places, why isn't it a good idea here?